src/zjs_util.c

```diff
@@ -21,7 +25,50 @@ void *zjs_malloc_with_retry(size_t size)
     }
     return ptr;
 }
+#ifdef ZJS_TRACE_MALLOC
+void zjs_print_mem_stats()
+{
+    for (int i = 0; i < max_list_size; i++) {
+        if (mem_list[i].ptr != NULL) {
+            ZJS_PRINT("index %i %s - %s:%d: %p\n", i, mem_list[i].file, mem_list[i].func, mem_list[i].line, mem_list[i].ptr);
+        }
+    }
+}
+
+void zjs_push_mem_stat(void *ptr, char *file, char *func, int line)
+{
+    int i = 0;
+    // Find an open spot
+    while (mem_list[i].ptr != NULL && i < max_list_size) {
+        i++;
+    }
+
+    if (i > max_list_size) {
+        ZJS_PRINT("zjs_push_mem_stat no memory stat slots available\n");
```

Comment:
Not so important since this is debug, but we've stopped putting function names in strings, to save space. The message is plenty unique w/o that.